src/main/java/io/cdap/plugin/zendesk/source/batch/ZendeskBatchSource.java

```diff
@@ -54,7 +55,11 @@ public ZendeskBatchSource(ZendeskBatchSourceConfig config) {
   @Override
   public void configurePipeline(PipelineConfigurer pipelineConfigurer) {
     FailureCollector failureCollector = pipelineConfigurer.getStageConfigurer().getFailureCollector();
-    config.validate(failureCollector);
+    try {
+      config.validate(failureCollector);
```

Comment:
Instead to throwing exception in validate, catch any exceptions thrown during validation and add a failure to the collector.
Also, don't do e.printStackTrace(); use logger instead.

Reply:
done